contents/java/JUnit5.md

```diff
@@ -0,0 +1,259 @@
+<frontmatter>
+  title: Introduction to JUnit 5
+  footer: footer.md
+  head: head.md
+  siteNav: mainNav.md
+  pageNav: 3
+</frontmatter>
+
+{{ navbar | safe }}
+
+<div class="website-content">
+
+# Introduction to Testing in Java with JUnit 5
+
+Author: [Lin Si Jie](https://github.com/sijie123)
+
+Reviewers: [Aadyaa Maddi](https://github.com/amad-person), [Marvin Chin](https://github.com/marvinchin)
+
+## What is JUnit 5?
+JUnit is an open-source automated testing framework for Java. Using its vast API library, you can easily write and run tests to ensure correctness in your software engineering project. JUnit 5 is the latest iteration of the JUnit project, supporting multiple new features in Java (such as Lambdas) and adopting a new <popover effect="fade" content="think: OOP for testing" placement="top">Extension-based model</popover> to make writing tests easier, cleaner, and more modular.
```

Comment:
Minimize unnecessary claims, such as _vast_, especially when the claim is subjective. The sentence can stand without it too. Same for _easily_.

Reply:
>JUnit 5 is the latest iteration ...

This will make the article incorrect as soon as there is a new version. Instead, just declare that this article is written to be compatible with JUnit 5. For similar reasons, the article title can do without `5` too.